diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -327,3 +327,5 @@ elseif(BUILD_CLIENT)
-        message(STATUS "crashpad_handler found in ${CRASHPAD_HANDLER_PROGRAM}")
-        file(COPY ${CRASHPAD_HANDLER_PROGRAM} DESTINATION ${BIN_OUTPUT_DIRECTORY})  
-        install(PROGRAMS ${CRASHPAD_HANDLER_PROGRAM} DESTINATION ${CMAKE_INSTALL_BINDIR})
+        if(CRASHPAD_HANDLER_PROGRAM)
+            message(STATUS "crashpad_handler found in ${CRASHPAD_HANDLER_PROGRAM}")
+            file(COPY ${CRASHPAD_HANDLER_PROGRAM} DESTINATION ${BIN_OUTPUT_DIRECTORY})  
+            install(PROGRAMS ${CRASHPAD_HANDLER_PROGRAM} DESTINATION ${CMAKE_INSTALL_BINDIR})
+        endif()
